README.md

```diff
@@ -49,7 +49,7 @@ The variables in the script itself can be overriden by environment variables. Th
 
 ## Usage
 * `spki init` - Initialize the PKI. This process first sets up the default Subject fields in the OpenSSL configuration files, then generates the Root CA, Intermediate CA, and a combined CA chain file. CRL's and OCSP certificates are also generated
```

Comment:
Can you add a note here about the automation you've added with env vars? We should list the necessary var names